scripts/execute.py

```diff
@@ -4,32 +4,32 @@
 import SS
 import TPI
 
-# model parameters
-S = int(40)
-E = 20
-T = 4 * S
-annual_beta = 0.9
-beta = annual_beta ** (40 / S)
-sigma = 3.0
-chi_n = 1.0 * np.ones(S)
-l_tilde = 1.0   # per period endowment for agent
-b_s = 0.6
-upsilon = 1.5
+# set model parameters
+sigma = 1.5  # CRRA
+beta = 0.8  # discount rate
+alpha = 0.3   # capital share of output
+delta = 0.1  # rate of depreciation
+A = 1.0  # TFP
+T = 20  # number of periods until SS
 
-# Firm 
-A = 1.0
-annual_delta = 0.05
-delta = 1 - ((1 - annual_delta) ** (40 / S))
-alpha = 0.5
+# parameter for convergence of GE loop
+xi = 0.1
+
+# Solving for SS
+'''
+Provided a guess of w = 0.05. Request to cross check if there is 
+any better guess that can be used
+
+We still need to add the TPI file and then update its execution here
+'''
+
+ss_params = (beta, sigma, alpha, A, delta, xi)
+r_init = 1 / beta - 1
+w_init = 0.5 # Not sure if there is a possible way to guess
```

Comment:
Why do you need a `w` guess?  If guess `r`, don't you know the `w` that is consistent with that?

Reply:
I am looking at the SS file and w_init as the input

But let me make sure once again

Reply:
It does seem like we need an initial guess of r and w to solve the HH problem
--> which gives b and n

Reply:
@devashish2019 But see the SS solution - `w` is found within the while loop.  Only `r_init` is passed to the `SS.solve_ss()` function.

Reply:
@jdebacker I am looking at the SS file which is on the master and it has w_init as an argument.

Reply:
The while look starts with the initial w guess (w = w_init) which is outside the for loop